### DISCUSSION TITLE
把用到的 Prompt 改成英文会不会节省 Token 的用量

### DISCUSSION THREAD

**Discussion by eastack** [nextai-translator/nextai-translator#264] [Q&A] [answered]
[OpenAI Tokenizer](https://platform.openai.com/tokenizer)
![image](https://user-images.githubusercontent.com/17524858/224865795-59cd8a25-eb1e-4d83-a87a-f5b38fd0efa9.png)

![image](https://user-images.githubusercontent.com/17524858/224865884-8b7358b3-23d4-4c5a-af2d-dfe033681495.png)

**Comment by yetone** [comment, marked as answer]
之前测过，用英文的话有时候会返回英文，即使强烈指定用中文返回
